builder/nunjucks/index.njk

```diff
@@ -222,7 +222,7 @@
                 Markup
               {% endif %}
             </summary>
-            <pre class="hljs"><code class="language-twig">{{ section.markup|hljs('twig') }}</code></pre>
+            <pre class="hljs"><code class="language-twig">{% raw %}{{ section.markup|hljs('twig') }}{% endraw %}</code></pre>
```

Comment:
Nunjunks tag `raw` is not equivalent to Twig filter `raw`.
In Nunjucks, the filter to not escape sting is named `safe`.
`{{ section.markup|hljs('twig')|safe }}`